.env

```diff
@@ -2,4 +2,8 @@ NODE_LOCAL_PORT=3000
 MONGODB_URI='mongodb://127.0.0.1:27017'
 DB_NAME= 'volunet'
 
-DEBUG = app:*
\ No newline at end of file
+DEBUG = app:*
+
+GOOGLE_CLIENT_ID=686293049369-neqjv8uibtdmjik1acmc7v616t67vgg0.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=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
+GOOGLE_CALLBACK_URL=http://localhost:3000/oauth2/redirect/google
```

Comment:
These are secret! they shouldn't be on a public repo on GitHub! someone can use them.

Reply:
Even though this was in .gitignore, apparently it was still being tracked. So "git rm .env --cached" is used to untrack it, recommend it for everyone.